cluster/local/kind.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+
+set -e
+
+scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# shellcheck disable=SC1090
+projectdir="${scriptdir}/../.."
+
+# get the build environment variables from the special build.vars target in the main makefile
+eval $(make --no-print-directory -C ${scriptdir}/../.. build.vars)
+
+BUILD_IMAGE="${BUILD_REGISTRY}/${PROJECT_NAME}-amd64"
+MINIKUBE_IMAGE="${DOCKER_REGISTRY}/${PROJECT_NAME}:master"
+DEFAULT_NAMESPACE="crossplane-system"
+
+function copy_image_to_cluster() {
+    local build_image=$1
+    local final_image=$2
+    docker tag "${build_image}" "${final_image}"
+    kind load docker-image "${final_image}"
+    echo "Tagged image: ${final_image}"
+}
+
+# Deletes pods with application prefix. Namespace is expected as the first argument
+function delete_pods() {
+    for pod in $(kubectl get pods -n "$2" -l "app=$1" --no-headers -o custom-columns=NAME:.metadata.name); do
+        kubectl delete pod "$pod" -n "$2"
+    done
+}
+
+# current kubectl context == minikube, returns boolean
+function check_context() {
+    if [ "$(kubectl config view 2>/dev/null | awk '/current-context/ {print $NF}')" = "kind-kind" ]; then
+        return 0
+    fi
+
+    return 1
+}
+
+# configure minikube
+KUBE_IMAGE=${KUBE_IMAGE:-"kindest/node:v1.15.11@sha256:6cc31f3533deb138792db2c7d1ffc36f7456a06f1db5556ad3b6927641016f50"}
```

Comment:
It looks like you hardcode 1.15 image below so I don’t think setting this variable would actually change anything